pages/_app.tsx

```diff
@@ -19,7 +19,7 @@ function MyApp({ Component, pageProps, apollo }: IProps) {
   const session = useSession()
   useEffect(() => {
     if (process.env.NODE_ENV === 'production') {
```

Comment:
should be.... `if (process.env.NODE_ENV === 'production && process.env.POSTHOG....)`